templates/cloudflare-workers/vite.config.ts

```diff
@@ -0,0 +1,19 @@
+import { defineConfig } from "vite";
+import {
+  vitePlugin as remix,
+  cloudflareDevProxyVitePlugin,
+} from "@remix-run/dev";
+import tsconfigPaths from "vite-tsconfig-paths";
+
+export default defineConfig((env) => ({
+  plugins: [cloudflareDevProxyVitePlugin(), remix(), tsconfigPaths()],
+  ...(env.isSsrBuild && {
+    resolve: {
+      conditions: ["workerd", "worker", "browser"],
+      mainFields: ["browser", "module", "main"],
+    },
```

Comment:
These could be specified within the `ssr` field so that the `env.isSsrBuild` check is not needed

```suggestion
  ssr: {
    resolve: {
      conditions: ["workerd", "worker", "browser"],
      mainFields: ["browser", "module", "main"],
    },
  },
```

Reply:
`mainFields` isn't available on `ssr.resolve` 🤔

Reply:
TIL!

Reply:
I managed to clean this up to avoid the `env.isSsrBuild` check.